products/driver_license/response_v1/complete.json

```diff
@@ -0,0 +1,298 @@
+              0.372
+            ]
+          ],
+          "value": "1957-02-01"
+        },
+        "dd_number": {
+          "page_id": 0,
+          "polygon": [
+            [
+              0.383,
+              0.85
+            ],
+            [
+              0.658,
+              0.85
+            ],
+            [
+              0.658,
+              0.989
+            ],
+            [
+              0.383,
+              0.989
+            ]
+          ],
+          "value": "DD1234567890123456"
+        },
+        "expiry_date": {
+          "page_id": 0,
+          "polygon": [
+            [
+              0.445,
+              0.635
+            ],
+            [
+              0.631,
+              0.635
+            ],
+            [
+              0.631,
+              0.68
+            ],
+            [
+              0.445,
+              0.68
+            ]
+          ],
+          "value": "2018-02-01"
+        },
+        "first_name": {
+          "page_id": 0,
+          "polygon": [
+            [
+              0.384,
+              0.452
+            ],
+            [
+              0.502,
+              0.452
+            ],
+            [
+              0.502,
+              0.491
+            ],
+            [
+              0.384,
+              0.491
+            ]
+          ],
+          "value": "Jelani"
+        },
+        "id": {
+          "page_id": 0,
+          "polygon": [
+            [
+              0.723,
+              0.259
+            ],
+            [
+              0.914,
+              0.259
+            ],
+            [
+              0.914,
+              0.311
+            ],
+            [
+              0.723,
+              0.311
+            ]
+          ],
+          "value": "D12345678"
+        },
+        "issued_date": {
+          "page_id": 0,
+          "polygon": [
+            [
+              0.736,
+              0.635
+            ],
+            [
+              0.922,
+              0.635
+            ],
+            [
+              0.922,
+              0.68
+            ],
+            [
+              0.736,
+              0.68
+            ]
+          ],
+          "value": "2013-01-10"
+        },
+        "issuing_authority": {
+          "page_id": 0,
+          "polygon": [],
+          "value": null
+        },
+        "last_name": {
+          "page_id": 0,
+          "polygon": [
+            [
+              0.384,
+              0.395
+            ],
+            [
+              0.521,
+              0.395
+            ],
+            [
+              0.521,
+              0.437
+            ],
+            [
+              0.384,
+              0.437
+            ]
+          ],
+          "value": "Sample"
+        },
+        "mrz": {
+          "page_id": 0,
+          "polygon": [],
+          "value": null
+        },
+        "place_of_birth": {
+          "page_id": 0,
+          "polygon": [],
+          "value": null
+        },
```

Comment:
Can you give these a value for the tests, please? same thing for summary_full.rst

Reply:
done